src/main/io/gps_ublox.c

```diff
@@ -840,13 +879,13 @@ STATIC_PROTOTHREAD(gpsConfigure)
     configureSBAS();
     ptWaitTimeout((_ack_state == UBX_ACK_GOT_ACK || _ack_state == UBX_ACK_GOT_NAK), GPS_CFG_CMD_TIMEOUT_MS);
 
-    // Enable GALILEO
-    if (gpsState.gpsConfig->ubloxUseGalileo && capGalileo) {
-        // If GALILEO is not supported by the hardware we'll get a NAK,
-        // however GPS would otherwise be perfectly initialized, so we are just waiting for any response
-        gpsSetProtocolTimeout(GPS_SHORT_TIMEOUT);
-        configureGalileo();
-        ptWaitTimeout((_ack_state == UBX_ACK_GOT_ACK || _ack_state == UBX_ACK_GOT_NAK), GPS_CFG_CMD_TIMEOUT_MS);
+    // Configure GNSS for M8N and later
+    if ((gpsState.gpsConfig->provider == GPS_UBLOX) || (gpsState.hwVersion < 70000)) {
```

Comment:
This looks wrong. `hwVersion < 70000` will match only M6N.

Reply:
I'll look at that. It's still very much WIP ...